File: 2015 maj/c++/1.cpp
#include <iostream>
#include <fstream>
using namespace std;


int main()
{
    ifstream in("liczby.txt");

    string s;
    int ans = 0;
    for (int line = 1; line <= 1000; line++)
    {
        in >> s;
        int cur = 0;
        for (int i = 0; i < s.size(); i++)
            if (s[i] == '0')
                cur++;
        if (cur > s.size() - cur)
            ans++;
    }
    cout << ans << '\n';
}

File: 2015 maj/python/1.py
with open("liczby.txt") as f:
    s = f.read().strip().split('\n')

ans = 0
for liczba in s:
    zero, one = 0, 0
    for i in liczba:
        if i == '0':
            zero += 1
        else:
            one += 1
    if zero > one:
        ans += 1
print(ans)